src/app/donut/donut.component.ts

```diff
@@ -90,7 +93,7 @@ export class DonutComponent implements OnInit, OnChanges {
         y: -30,
       },
       subtitle: {
-        text: 'montlhy payments',
+        text: data === this.data ? this.middleLabel[0] : this.middleLabel[1],
```

Comment:
maybe this could be passed as another binding from parent component, not to be hardcoded in 'donut'

Reply:
like @Input() middleLabel: text;